AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/edit-work-hours/vho-work-hours-non-availability-table/vho-work-hours-non-availability-table.component.ts

```diff
@@ -340,6 +347,7 @@ export class VhoWorkHoursNonAvailabilityTableComponent implements OnInit, CanDea
 
         this.nonWorkHours.push(editVhoNonAvailabilityWorkHoursModel);
         this.onStartDateBlur(editVhoNonAvailabilityWorkHoursModel);
+        this.switchToEditMode();
```

Comment:
this needs to be at the start of the function, otherwise it causes a bug when clicking cancel (the non-availability being added goes into the table, even though it hasn't been added yet)

Reply:
moved to top